### ISSUE TITLE
[Bug] Wither Head Trajectories

### ISSUE THREAD

**Issue by RickyTheRacc** [MeteorDevelopment/meteor-client#2127]
### Describe the bug

If you turn on trajectories for already fired projectiles, wither heads will still show them drooping down despite not being affected by gravity

### Steps to reproduce

```bash
Enable tracjectories
Turn on "Fired Projectiles" and "Accurate"
Spawn a wither and get it to shoot at you
Notice the wither heads are displayed as having dropoff
```

### Meteor Version

1441

### Minecraft Version

1.18.2

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

**Comment by shittass** [comment]
MEANINGLESS

**Comment by MineGame159** [comment]
Fixed in e4bdb373ecaec35691e73a27675c1b8805f1de45.
